routes/catalog.js

```diff
@@ -46,11 +53,21 @@ catalogRouter.get('/', async (req, res) => {
     } else {
       params.push('');
     }
+    params.push(limit);
+    params.push(offset);
 
     query += ' ORDER BY title ASC';
+    
+    let countQuery = 'SELECT COUNT(*) ' + query + ';';
```

Comment:
🚫 **[eslint]** <[prefer-const](https://eslint.org/docs/rules/prefer-const)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'countQuery' is never reassigned. Use 'const' instead.
```suggestion
    const countQuery = 'SELECT COUNT(*) ' + query + ';';
```